StringCalculatorKata.Test/StringCalculatorTest.cs

```diff
@@ -0,0 +1,75 @@
+﻿using FluentAssertions;
+using AutoFixture;
+
+namespace StringCalculatorKata.Test
+{
+    public class StringCalculatorTest
+    {
+        [Theory]
+        [InlineData("", 0)]
+        [InlineData("1", 1)]
+        [InlineData("1,1", 2)]
+        public void AddingUpToTwoNumbers(string numbers, int actualSum)
```

Comment:
the name of the test should be in this format: serviceName_ShouldResult_WhenCase
please rename all tests to match it